skops/io/tests/test_persist.py

```diff
@@ -785,11 +787,13 @@ def test_loads_from_str():
 
 
 def test_get_instance_unknown_type_error_msg():
-    state = get_state(("hi", [123]), None)
```

Comment:
why do we need to remove `None`? We should be able to persist `None`.

Reply:
Oh sorry, I've done this for clarity

We now do essentially `state = get_state(("hi", [123]), SaveContext(None))`

The reason I did this is that `None` in the original line is the `zipfile` param to memoize certain objects, not something that actually gets persisted. I thought maybe moving it to 3 lines was clearer what was going on.

Reply:
Oh my bad, I just misread the parentheses, lol